modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/RankFeaturesFieldMapper.java

```diff
@@ -107,7 +111,14 @@ public ValueFetcher valueFetcher(SearchExecutionContext context, String format)
 
         @Override
         public Query termQuery(Object value, SearchExecutionContext context) {
-            throw new IllegalArgumentException("Queries on [rank_features] fields are not supported");
+            return TERM_QUERY_BUILDER.toQuery(name(), indexedValueForSearch(value), true);
```

Comment:
Nit: it looks as simple to me to call `FeatureField.newLinearQuery(name(), indexedValueForSearch(value), DEFAULT_BOOST)` directly instead of going through this abstraction?